lib/teleterm/clusters/cluster_auth.go

```diff
@@ -325,8 +328,19 @@ func (p *pwdlessLoginPrompt) PromptPIN() (string, error) {
 }
 
 // PromptTouch prompts the user for a security key touch.
-func (p *pwdlessLoginPrompt) PromptTouch() error {
-	return trace.Wrap(p.Stream.Send(&api.LoginPasswordlessResponse{Prompt: api.PasswordlessPrompt_PASSWORDLESS_PROMPT_TAP}))
+func (p *pwdlessLoginPrompt) PromptTouch() (wancli.TouchAcknowledger, error) {
+	return p.acknowledgeTouch, trace.Wrap(p.Stream.Send(&api.LoginPasswordlessResponse{Prompt: api.PasswordlessPrompt_PASSWORDLESS_PROMPT_TAP}))
+}
+
+func (p *pwdlessLoginPrompt) acknowledgeTouch() error {
+	// TODO: Send touch acknowledgement if worth the effort, this is not
```

Comment:
Add yourself to the TODO, so it's clear whom to ask about it.

I see that @ravicious is in the reviewers set, but tagging him here regardless as an FYI.

```suggestion
	// TODO(nklaassen): Send touch acknowledgement if worth the effort, this is not
```

Reply:
Alright, tagged myself so I can be asked about it. I have a feeling it won't be worthwhile for Connect at the moment, this streaming RPC seems impossible to change, and we'd have to build a another one to get it working for passwordfull MFA logins as well.

Reply:
When it comes to gRPC communication between Connect and tsh daemon, we don't have to worry about breaking older clients as both the client and the server are always updated in tandem.

It'd be super handy to have this in Connect as Connect suffers from the same problem as tsh so thanks for adding the API for that. Implementing this would require some refactoring when it comes to the regular and passwordless login flows. We don't have time to take care of this at the moment but I'll add an issue to remember about this feature.